lib/publiccloud/provider.pm

```diff
@@ -478,7 +478,6 @@ sub terraform_apply {
         my $sap_media = get_required_var('HANA');
         my $sap_regcode = get_required_var('SCC_REGCODE_SLES4SAP');
```

Comment:
Kind question: given the declared scope in PR header, are parameters like this or `*HANA` too still needed instead?

Reply:
I hope to be able to remove more code from this file in the near future, But it is beyond the purpose of the associated ticket.